### PR TITLE
Update README.md; Add Community section

### PR DESCRIPTION
**Pull request by alolita**
Listed maintainers, approvers and triager for OTel Swift project repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,34 @@ The package includes some example projects with basic functionality:
...
-We are also available in the [#otel-swift](https://cloud-native.slack.com/archives/C01NCHR19SB) channel in the [CNCF slack](https://slack.cncf.io/). Please join us there for further discussions.
+We have a weekly SIG meeting! See the [community page](https://github.com/open-telemetry/community#swift-sdk) for meeting details and notes.
+
+We are also available in the [#otel-swift](https://cloud-native.slack.com/archives/C01NCHR19SB) channel in the [CNCF slack](https://slack.cncf.io/). Please join us there for OTel Swift discussions.
+
+### Community members
+
+#### Maintainers ([@open-telemetry/swift-maintainers](https://github.com/orgs/open-telemetry/teams/swift-maintainers))
+
+- [Ignacio Bonaforte](https://github.com/nachob), Crowdstrike
```
Comment (nachoBonafonte) [inline README.md:103]:
```suggestion
- [Ignacio Bonaforte](https://github.com/nachob), Independent
```
I am doing this personally not as part of my company duties

File: README.md [hunk 2]
```diff
@@ -88,7 +88,34 @@ The package includes some example projects with basic functionality:
...
+- [Bryce Buchanan](https://github.com/bryce-b), Elastic
+
+*Find more about the maintainer role in the [community repository](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer).*
+
+#### Approvers ([@open-telemetry/swift-approvers](https://github.com/orgs/open-telemetry/teams/swift-approvers))
+
+- [Vinod Vidiyer](https://github.com/vvydier)), Independent
```
Comment (nachoBonafonte) [inline README.md:110]: @vvydier check if you want to appear as independent or as part of your company

File: README.md [hunk 3]
```diff
@@ -88,7 +88,34 @@ The package includes some example projects with basic functionality:
...
+- [Austin Emmons](https://github.com/atreat), Independent
```
Comment (nachoBonafonte) [inline README.md:111]: @atreat check if you want to use your company instead of Independent here

**Review by nachoBonafonte — CHANGES_REQUESTED** [review]

**Comment by nachoBonafonte** [comment]
Looks great to me, just a small change on my side and checking Vinod and Austin preferences for their company name

**Comment by alolita** [comment]
@vvydier confirmed current contributor status as Independent.

**Review by bryce-b — APPROVED** [review]

**Pull request merged**